index.html

```diff
@@ -0,0 +1,788 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Page Title</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="css/main.css" />
+    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <nav>
+        <div class="space"><span>&#9776;</span> <span><i class="fa fa-search"></i></span> <a href="#"> SPACE &
+                COSMOS</a></div>
+        <div class="newyork"> <a href="#"> <img src="img/newyork.PNG" alt=""></a></div>
+        <div class="buttons">
+            <button type="submit" class="subscribe">SUBSCRIBE NOW</button> <button type="submit" class="login">LOG
+                IN</button>
+
+        </div>
+    </nav>
+
+    <br />
+    <br />
+    <header class="page-title">
+        Space Ripples Reveal
```

Comment:
**[OPTIONAL]**
It's best practice to use the main tag for the most relevant content on your page. In this case, the main article.